pandas/tools/pivot.py

```diff
@@ -395,6 +395,16 @@ def crosstab(index, columns, values=None, rownames=None, colnames=None,
         Add row/column margins (subtotals)
     dropna : boolean, default True
         Do not include columns whose entries are all NaN
+    normalize : boolean, {'all', 'index', 'columns'}, or {0,1}, default False
+        Normalize by dividing all values by the sum of values.
+
+        - If passed `all` or `True`, will normalize over all values.
```

Comment:
Can you put `'` around 'all'? Then the difference with True (string vs bool) is more clear I think